app/Console/Commands/FixPermissions.php

```diff
@@ -88,8 +89,9 @@ public function handle(): int
 	private function fixPermissionsRecursively(string $path): void
 	{
 		try {
-			$actualPerm = fileperms($path);
-			if ($actualPerm === false) {
+			try {
+				$actualPerm = fileperms($path);
+			} catch (FilesystemException) {
```

Comment:
Why adding a separate `catch`-block here when we return anyway? Why not using the `try`-`catch`-structure we already have and catching this exception at the end of the whole block? This would avoid an unnecessary level of indentation.

Reply:
fixed [ae607b8](https://github.com/LycheeOrg/Lychee/pull/1469/commits/ae607b8418eb244707f42e45b970298328d2eb6e)